### PR TITLE
feat - input component

### PR DESCRIPTION
**Pull request by prachi-sahani**
Created input component
Types of inputs included:
 - outlined textarea
 - outlined input
 - inputs with valid and invalid messages

input

### CODE REVIEW

**Review by FarhanMobashir — COMMENTED** [review]
All good ✅

**Review by abhij1607 — APPROVED** [review]
LGTM👍

File: css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+.input-group, .textarea-group{
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+}
+
+.input-outline, .textarea-outline{
+    margin-top: 3px;
```
Comment (MaruthiPothuganti) [inline css/components/input.css]: Suggestion: can use shorthand property for margin

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: use of short hand property

File: css/components/input.css (hunk 1 above)
Comment (prachi-sahani) [reply css/components/input.css]: Done 👍🏻

**Pull request merged**